src/App.js

```diff
@@ -12,6 +12,7 @@ class App extends Component {
     title: 'My Bank Account',
     currency: '€', // &eur; !!!
     transactions: [],
+    filterAmount: null
```

Comment:
Salviamo il filtro corrente nello stato del componente piu' in alto nella gerarchia, in modo che sia facile poi filtrare avendo la lista della transazioni e il filtro nello stesso posto (questo e' un pattern che in genere funziona molto bene)